### DISCUSSION TITLE
Source current sign

### DISCUSSION THREAD

**Discussion by Abbas111977** [SpiceSharp/SpiceSharp#197] [Q&A] [answered]
Hello Sir,
               I would like to know why the current in the example page has -ve sign ? shouldn't it be +ve when the source +1 V ? 
[https://spicesharp.github.io/SpiceSharp/articles/tutorials/gettingstarted.html](url)

**Comment by svenboulanger** [comment, marked as answer]
This is just a convention. Current is defined from the positive to the negative terminal.
